lms/djangoapps/grades/tests/test_scores.py

```diff
@@ -47,20 +51,20 @@ class TestGetScore(TestCase):
     display_name = 'test_name'
     location = 'test_location'
 
-    SubmissionValue = namedtuple('SubmissionValue', 'exists, weighted_earned, weighted_possible')
-    CSMValue = namedtuple('CSMValue', 'exists, raw_earned, raw_possible')
+    SubmissionValue = namedtuple('SubmissionValue', 'exists, points_earned, points_possible, created_at')
```

Comment:
Because that's what the real submission api gives back.  These are essentially mocks.  I was mocking wrong, and hence getting wrong results.